elide-core/src/main/java/com/yahoo/elide/core/QueryDetail.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core;
+
+import lombok.Value;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * QueryDetail implementation class. Stores the model name (root entity class) and
+ * the queryText that the underlying datastore will eventually run
+ */
+@Value
+public class QueryDetail {
```

Comment:
You can add a lombok builder interface alternatively instead of defining all of the constructors.

Reply:
Fixed in most recent commit